src/app/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { FileHandle } from '@eco-news-models/create-news-interface';
+import { ImageCroppedEvent } from 'ngx-image-cropper';
+import { EditProfileService } from '@global-user/services/edit-profile.service';
+import { MatDialog } from '@angular/material';
+import { ErrorComponent } from '@errors/error/error.component';
+
+@Component({
+  selector: 'app-edit-photo-pop-up',
+  templateUrl: './edit-photo-pop-up.component.html',
+  styleUrls: ['./edit-photo-pop-up.component.scss']
+})
+export class EditPhotoPopUpComponent implements OnInit {
+  public avatarImg: string;
+  public cancelButton = './assets/img/profile/icons/cancel.svg';
+  public selectedPhoto = false;
+  public selectedFile: File = null;
+  public selectedFileUrl: string;
+  public files: FileHandle[] = [];
+  public isWarning = false;
+  private croppedImage: string;
+
+  constructor(private matDialogRef: MatDialogRef<EditPhotoPopUpComponent>,
+              private dialog: MatDialog,
+              private editProfileService: EditProfileService,
+              @Inject(MAT_DIALOG_DATA) public data) { }
+
+  ngOnInit() {
+    this.setUserAvatar();
+  }
+
+  private setUserAvatar() {
+    this.avatarImg = this.data.img;
+  }
+
+  public closeEditPhoto(): void {
+    this.matDialogRef.close();
+  }
+
+  public onSelectPhoto(event): void {
+    this.isWarning = false;
+    this.selectedFile = event.target.files[0] as File;
+    const reader: FileReader = new FileReader();
+    reader.readAsDataURL(this.selectedFile);
+    reader.onload = (ev) => this.handleFile(ev);
+  }
+
+  private handleFile(event): void {
+    this.selectedFileUrl = event.target.result;
+    this.files[0] = { url: this.selectedFileUrl, file: this.selectedFile };
+    if (!this.isWarning && typeof this.selectedFile !== 'undefined') {
```

Comment:
Is it neccessary to write "typeof this.selectedFile !== 'undefined'"? Try to rewrite it just as "this.selectedFile"

Reply:
it’s all right here  - in this condition, according to [Style Guide](https://github.com/airbnb/javascript#comparison--shortcuts)